### PR TITLE
[#583] @JsonbCreator uses PropertyNamingStrategy.

### PR DESCRIPTION
**Pull request by bmarwell**
This *might* work, but I am not sure about what to pass in `src/main/java16/org/eclipse/yasson/internal/ClassMultiReleaseExtension.java`.

### CODE REVIEW

File: src/main/java16/org/eclipse/yasson/internal/ClassMultiReleaseExtension.java [hunk 1]
```diff
@@ -50,7 +51,7 @@ static JsonbCreator findCreator(Class<?> clazz,
                                     AnnotationIntrospector introspector) {
         if (clazz.isRecord()) {
             if (declaredConstructors.length == 1) {
-                return introspector.createJsonbCreator(declaredConstructors[0], null, clazz);
+                return introspector.createJsonbCreator(declaredConstructors[0], null, clazz, pns -> pns);
```
Comment (Verdent) [inline src/main/java16/org/eclipse/yasson/internal/ClassMultiReleaseExtension.java:54]: I might say, add PropertyNamingStrategy parameter. This class is in internal package so it should be fine to change it. That should fix it even for constructors without creator annotation

**Review by Verdent — CHANGES_REQUESTED** [review]

**Comment by Verdent** [comment]
Watch out for those copyright failures :-) It should now be 2023.

**Comment by bmarwell** [comment]
> Watch out for those copyright failures :-) It should be 2023 now.

Actually the plugin does not check commit date, but author date. As I amended the previous commit, it must be 2022 according to the plugin. Unexpected, but that is what it takes to make the plugin happy. :)

**Comment by Verdent** [comment]
Oh it seems you are having typo there :-)

**Comment by bmarwell** [comment]
Sorry, broken keyboard (spilled soda over it) 😆

**Comment by jhanders34** [comment]
Any more progress on this PR?

**Comment by Verdent** [comment]
Oh, it seems it is failing on incorrectly separated imports. Otherwise, I think it is OK.

**Comment by bmarwell** [comment]
> Oh, it seems it is failing on incorrectly separated imports. Otherwise, I think it is OK.

Sigh, sorry! I am too used to checking in IDE config 😞  will fix this ASAP.

**Comment by Verdent** [comment]
Perfect, thank you for this contribution @bmarwell

**Review by Verdent — APPROVED** [review]

**Pull request merged**